vnext/Microsoft.ReactNative.Cxx/CoreApp.h

```diff
@@ -0,0 +1,62 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+#pragma once
+
+struct RNCoreApp {
+  /// default value: L"index.windows"
+  wchar_t const *jsBundleFile;
+  /// default value: L"ms-appx:///Bundle/"
+  wchar_t const *bundleRootPath;
+  /// Provide the name of your React component here
+  wchar_t const *componentName;
+
+  /// default value: true
+  bool useWebDebugger;
```

Comment:
Can we just set the default values here so we don't risk warnings later if someone doesn't initialize the struct? \#CodeQL

Reply:
sadly no as this needs to be usable from C (hence why using typedef below, etc.)